android/java/org/chromium/chrome/browser/toolbar/top/BraveToolbarLayoutImpl.java

```diff
@@ -704,10 +704,8 @@ public void addMediaToPlaylist() {
         if (mPlaylistService == null || currentTab == null) {
             return;
         }
-        org.chromium.url.mojom.Url contentUrl = new org.chromium.url.mojom.Url();
-        contentUrl.url = currentTab.getUrl().getSpec();
-        mPlaylistService.addMediaFilesFromPageToPlaylist(
-                ConstantUtils.DEFAULT_PLAYLIST, contentUrl, true);
+        mPlaylistService.addMediaFilesFromActiveTabToPlaylist(
+                ConstantUtils.DEFAULT_PLAYLIST, true, addedItems -> {});
```

Comment:
I don't understand why you're passing an empty callback here

Reply:
I mean it certainly seems like this method should have a callback, but we are apparently ignoring the result here

Reply:
Asked @deeppandya what should be called. I'll update it when I get response.

Reply:
@sangwoo108  Let's connect in DM.